site/src/App/Documentation/ComponentRoute.tsx

```diff
@@ -70,16 +67,14 @@ export const ComponentRoute = ({
 
           const codeAsString =
             Example && !code
-              ? cleanCodeSnippet(
-                  reactElementToJSXString(Example({ id: 'id', handler }), {
-                    useBooleanShorthandSyntax: false,
-                    showDefaultProps: false,
-                    showFunctions: false,
-                    filterProps: ['onChange', 'onBlur', 'onFocus'],
-                  }),
-                )
+              ? reactElementToJSXString(Example({ id: 'id', handler }), {
+                  useBooleanShorthandSyntax: false,
+                  showDefaultProps: false,
+                  showFunctions: false,
+                  filterProps: ['onChange', 'onBlur', 'onFocus'],
+                })
               : code
-              ? cleanCodeSnippet(dedent(code))
+              ? dedent(code)
```

Comment:
Do we still need `dedent` if prettier is running?

Reply:
Good catch 👍